web/app/view/GroupDialog.js

```diff
@@ -40,5 +40,18 @@ Ext.define('Traccar.view.GroupDialog', {
             displayField: 'name',
             valueField: 'id'
         }]
-    }
+    },
+
+    buttons: [{
```

Comment:
Should we move this to the base class. Most (or all?) models have attributes now.